pup-0002.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
+
+## Motivation
+
+Most current Pulp developers find Google Style docstrings to be more readable than the default
+Sphinx style docstrings.
+
+## Details
+
+Upon acceptance, these changes will be documented in the Pulp style guide.
+
+### Style Choices
+
+Docstrings should follow the conventions documented in the
+[Google style guide](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)
+except where Pulp specific additions or exceptions apply.
+
+Additionally, docstrings must be parsable by the
```

Comment:
I think this can be moved to a section called "Requirements". We require that docstrings be parsable by Sphinx (not necessarily Napoleon).